openclaw_hooks/nima-recall-live/lazy_recall.py

```
@@ -1,468 +1,438 @@
+    args = parser.parse_args()
     
-    query = sys.argv[1]
     current_affect = None
-    
-    # Parse current affect if provided
-    if len(sys.argv) > 2:
+    if args.affect:
         try:
-            current_affect = json.loads(sys.argv[2])
+            current_affect = json.loads(args.affect)
         except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)